external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -125,50 +125,52 @@ The `from_existing` value when set provides the basis of linked data; it allows
 
-| **field name** | f**description**                                                                                                                                                                                                                                                                     | **default value** |
-| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------- |
-| `type`         | The type of the column, provide `"measure column"` for the measure column type or `"unit column"` for the unit column (Required)                                                                                                                                                     | *dimension*       |
-| `values`       | If basic units/measures are desired, a boolean value of `true` is used to signify to csvcubed to create units/measures from values of this column; otherwise values is a dictionary which defines the units/measures using the notion from [Measures and Units](#Measures and Units) | `true`            |
+| **field name**           | f**description**                                                                                                                                                                                                                                                                     | **default value** |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `type`                   | The type of the column, provide `"measure column"` for the measure column type or `"unit column"` for the unit column (Required)                                                                                                                                                     | *dimension*       |
+| `values`                 | If basic units/measures are desired, a boolean value of `true` is used to signify to csvcubed to create units/measures from values of this column; otherwise values is a dictionary which defines the units/measures using the notion from [Measures and Units](#Measures and Units) | `true`            |
+| `describes_observations` | (Unit column only) Associates observation values with the relevant unit. This is only necessary for [pivoted shape data sets](../shape-data/pivoted-shape.md) with multiple observation value columns.                                                                                                                 | *none*            |
```

Comment:
It's associating the *units column* with the relevant observation values, not an individual unit.

Reply:
Done